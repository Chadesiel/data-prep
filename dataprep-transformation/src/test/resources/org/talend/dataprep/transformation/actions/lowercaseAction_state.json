{
  "actions": [
    {
      "action": "lowercase",
      "parameters": {
        "column_id": "state",
        "scope": "column"
      }
    }
  ]
}