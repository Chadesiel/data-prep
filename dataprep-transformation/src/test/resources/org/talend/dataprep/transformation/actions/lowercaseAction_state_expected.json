{
  "records": [
    {
      "state": "nc",
      "tdpId": 1
    },
    {
      "state": "ny",
      "tdpId": 2
    },
    {
      "state": "az",
      "tdpId": 3
    },
    {
      "state": "ca",
      "tdpId": 4
    }
  ],
  "columns": [
    {
      "name": "state",
      "headerSize": 0,
      "type": "string",
      "quality": {
        "empty": 0,
        "invalid": 0,
        "valid": 4,
        "invalidValues": []
      },
      "id": "state",
      "statistics": {
        "count": 4,
        "valid": 4,
        "invalid": 0,
        "empty": 0,
        "max": "NaN",
        "min": "NaN",
        "mean": "NaN",
        "variance": "NaN",
        "duplicateCount": 0,
        "distinctCount": 4,
        "frequencyTable": [
          {
            "data": "nc",
            "occurrences": 1
          },
          {
            "data": "az",
            "occurrences": 1
          },
          {
            "data": "ny",
            "occurrences": 1
          },
          {
            "data": "ca",
            "occurrences": 1
          }
        ],
        "patternFrequencyTable": [
          {
            "pattern": "aa",
            "occurrences": 4
          }
        ],
        "quantiles": {
          "median": "NaN",
          "lowerQuantile": "NaN",
          "upperQuantile": "NaN"
        },
        "histogram": [],
        "textLengthSummary": {
          "minimalLength": 2.0,
          "maximalLength": 2.0,
          "averageLength": 2.0
        }
      },
      "domain": "US_STATE_CODE",
      "domainLabel": "US State Code",
      "domainFrequency": 0.0,
      "semanticDomains": [],
      "domainForced": false,
      "typeForced": false
    }
  ]
}